hatchery/src/world/event.rs

```diff
@@ -13,12 +13,23 @@ use std::ops::Deref;
 pub struct Receipt<T> {
     ret: T,
     events: Vec<Event>,
+    debug: Vec<String>,
```

Comment:
I'm not sure this should go in a receipt. Could you try and justify this choice?

Reply:
Mostly for verifying that debug works in the tests.

Reply:
This will be refactored as well with the Session changes

Reply:
It's also zero overhead really - apart from 3 integers :sweat_smile: